R/fledgling.R

```diff
@@ -177,8 +177,9 @@ read_fledgling <- function() {
   date <- read_date()
 
   news_and_preamble <- read_news()
+  news_and_preamble[["preamble"]] <- news_and_preamble[["preamble"]] %||% news_preamble()
```

Comment:
I moved the preamble "filling" here in order to be able to use `read_news()` for checking whether there's a preamble in the `NEWS.md` file.